### DISCUSSION TITLE
Display GiB or number of plots plotted per day

### DISCUSSION THREAD

**Discussion by Jonamac** [swar/Swar-Chia-Plot-Manager#845] [Ideas]
Hello,

Not sure if this has already been suggested but it'd be awesome if the program could automatically tell you how fast your system is running as far as GiB and/or plots per 24 hrs. 
So take the size of all the fully completed plots in the current job (in GiB) and divide by amount of time it took to finish those plots (including final transfer) and then multiply by 24.

I think this would be helpful for anyone using the advanced results option on chiacalculator.com and want to see more accurate results. It'd save a lot of time digging through the plotter log files.

**Comment by creamsoup** [comment]
it has yesterday and today's plot. not exactly past 24 hours but similar idea.
